### PR TITLE
docs: add some details about torcx interaction

### PR DESCRIPTION
**Pull request by lucab**

### CODE REVIEW

File: Documentation/torcx.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Torcx configuration managed by Tectonic
```
Comment (crawford) [inline Documentation/torcx.md]: torcx is always lowercase, right?

File: Documentation/torcx.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      {
+        "name": "docker",
+        "reference": "1.12"
+      }
+    ]
+  }
+}
+```
+
+The result of the example above is that the `docker:1.12` addon will be applied on the next machine boot.
```
Comment (crawford) [inline Documentation/torcx.md]: add-on

Comment (lucab) [reply Documentation/torcx.md]: We consistently use "addon" everywhere else:
 * https://github.com/coreos/torcx
 * https://github.com/coreos/docs/blob/master/torcx/what-is-torcx.md

File: Documentation/torcx.md (hunk 1 above)
Comment (lucab) [reply Documentation/torcx.md]: It doesn't have any strong casing. `tectonic-torcx` is always used lowercase as a binary name, while torcx follows normal casing rules depending on the context as it refers to a mix of project, binary, and general addon framework.

**Pull request merged**